korangar/src/loaders/model/mod.rs

```diff
@@ -107,7 +107,8 @@ impl ModelLoader {
                 None => [face.smooth_group, -1, -1],
                 Some(extras) if extras.len() == 1 => [face.smooth_group, extras[0], -1],
                 Some(extras) if extras.len() == 2 => [face.smooth_group, extras[0], extras[1]],
-                _ => panic!("more than three smoothing groups found"),
+                // TODO: add for more than two smoothing groups, default as no smoothing group
```

Comment:
We could use "SmallVec" in this case, where we optimize for 3 smooth groups and allow to grow more.